notifier.rst

```diff
@@ -426,6 +426,7 @@ Service          Package                               DSN
 `Engagespot`_    ``symfony/engagespot-notifier``       ``engagespot://API_KEY@default?campaign_name=CAMPAIGN_NAME``
 `Expo`_          ``symfony/expo-notifier``             ``expo://Token@default``
 `OneSignal`_     ``symfony/one-signal-notifier``       ``onesignal://APP_ID:API_KEY@default?defaultRecipientId=DEFAULT_RECIPIENT_ID``
+`Ntfy`_          ``symfony/ntfy-notifier``             ``ntfy://default/TOPIC``
```

Comment:
Please move above OneSignal, thanks

Reply:
I just rebased, I will review and add a 6.4 paragraph 😉